disco_aws_automation/disco_storage.py

```diff
@@ -195,16 +192,6 @@ def configure_storage(self,
         logging.debug("mapped %s to root partition", disk_names[current_disk])
         current_disk += 1
 
-        # Map the latest snapshot for this hostclass
-        if map_snapshot:
-            latest = self.get_latest_snapshot(hostclass)
-            if latest:
-                self.wait_for_snapshot(latest)
-                current_name = disk_names[current_disk]
-                bdm[current_name] = self.create_snapshot_bdm(latest, iops)
-                logging.debug("mapped %s to snapshot %s", current_name, latest.id)
-                current_disk += 1
-
         # Map extra disk
         if extra_disk:
```

Comment:
We can also first make sure we have the available device name for EBS volume, in this case xvdz. After that, we can then use the rest of device names for ephemeral disks.

I agree with your early comment that if there is no use cases for extra disk, we should remove the code for it.